homeassistant/components/websocket_api/commands.py

```diff
@@ -95,6 +100,42 @@ def forward_events(event):
     connection.send_message(messages.result_message(msg["id"]))
 
 
+@callback
+@decorators.websocket_command(
+    {
+        vol.Required("type"): "subscribe_bootstrap_integrations",
+    }
+)
+def handle_subscribe_bootstrap_integrations(hass, connection, msg):
+    """Handle subscribe bootstrap integrations command."""
+
+    @callback
+    def forward_bootstrap_integrations(message):
+        """Forward bootstrap integrations to websocket."""
+        connection.send_message(messages.result_message(msg["id"], message))
+
+    connection.subscriptions[msg["id"]] = async_dispatcher_connect(
+        hass, SIGNAL_BOOTSTRAP_INTEGRATONS, forward_bootstrap_integrations
+    )
+
+    connection.send_message(messages.result_message(msg["id"]))
+
+
+@callback
+@decorators.websocket_command(
+    {
+        vol.Required("type"): "unsubscribe_bootstrap_integrations",
+        vol.Required("subscription"): cv.positive_int,
+    }
+)
+def handle_unsubscribe_bootstrap_integrations(hass, connection, msg):
```

Comment:
We don't need this. We already store the unsub in line 117 and so the frontend can just cancel it.

Reply:
Yeah, just verified. The frontend automatically calls `unsubscribe_events` to cancel subscriptions https://github.com/home-assistant/home-assistant-js-websocket/blob/master/lib/connection.ts#L303-L307

Reply:
removed handle_unsubscribe_bootstrap_integrations